pkg/datastore/errors.go

```diff
@@ -19,7 +19,7 @@ func (err ErrNamespaceNotFound) NotFoundNamespaceName() string {
 
 // MarshalZerologObject implements zerolog object marshalling.
 func (err ErrNamespaceNotFound) MarshalZerologObject(e *zerolog.Event) {
```

Comment:
I actually think if we used `e` as the receiver, this would've been unlikely to happen. Maybe we should move towards that, too.